Packs/CommonScripts/Scripts/FileEnrichment/FileEnrichment.yml

```diff
@@ -0,0 +1,210 @@
+commonfields:
+  id: file-enrichment
+  version: -1
+args:
+- name: file_hash
+  description: 'Hash of the file. Supported types are: MD5, SHA1, SHA256, and SHA512.'
+  required: true
+- name: enrichment_brands
+  description: |-
+    Integrations brands to use for running the file reputation command. If not provided, the command will run for all available integrations that contain the file reputation command.
+    For multi-select, provide a comma-separated list of integration internal names. For example: "VirusTotal (API v3),WildFire-v2".
+  auto: PREDEFINED
+  predefined:
+  - 'VirusTotal (API v3)'
+  - 'WildFire-v2'
+  - 'Cortex Core - IR'
+- name: verbose
+  description: Whether to retrieve a human-readable entry for every command. When set to false, human-readable will be containing only summary of the final result.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  defaultValue: 'false'
+- name: external_enrichment
+  description: Whether to run additional external indicator enrichment commands. Set to true to enrich with information from the specified source brands. If set to false, only existing indicators in Threat Intelligence Module (TIM) will be retrieved.
```

Comment:
Whether to run additional external indicator enrichment commands. Set to true to enrich with information from the specified source brands. If set to false, only existing indicators in the Threat Intelligence Module (TIM) will be retrieved.